### DISCUSSION TITLE
sticky table header

### DISCUSSION THREAD

**Discussion by pete111** [tabler/tabler#1503] [General]
Hello, 
I describe my issue on https://preview.tabler.io/layout-navbar-sticky.html - please see Invoices at the bottom. I have the same page but with Invoices table only.
When I scroll down, I would like to have sticky header 

`NO. | INVOICE SUBJECT | CLIENT | VAT NO. ...`

 on the top. How can this be done please?

P.S. I tried to use sticky css, but it does not work because of overflow-x. But without overflow-x table is messed.

**Comment by cavasinf** [comment]
Maybe take a look at `people` list (bottom right) on https://preview.tabler.io/lists.html
Is that what you want to do?

**Comment by pete111** [comment]
`people` list seems very similar what I hoped for. But im my case my table has many columns overflowing in x axis  (to the right, outside visible area, but visible once user scrolls horizontaly to the right)

I tried to add more columns to `people` list with `<div class="col"><div class="">Sample text</div></div>` but it just make columns width very small on the screen.
